res/values/arrays.xml

```diff
@@ -2,74 +2,74 @@
 <resources>
   <string-array name="pref_layout_values">
     <item>none</item>
-    <item>azerty</item>
-    <item>qwerty</item>
-    <item>qwerty_pt</item>
-    <item>qwerty_es</item>
-    <item>qwerty_el</item>
-    <item>qwerty_ko</item>
-    <item>qwerty_lv</item>
-    <item>qwerty_no</item>
-    <item>qwerty_sv_se</item>
-    <item>qwerty_hu</item>
-    <item>qwerty_tr</item>
-    <item>qwerty_pl</item>
-    <item>qwerty_vi</item>
-    <item>ru_jcuken</item>
-    <item>qwertz</item>
-    <item>qwertz_cs</item>
-    <item>qwertz_de</item>
-    <item>qwertz_hu</item>
-    <item>qwertz_sk</item>
-    <item>bgph1</item>
-    <item>dvorak</item>
-    <item>colemak</item>
-    <item>neo2</item>
-    <item>bone</item>
-    <item>jcuken_ua</item>
-    <item>bangla</item>
-    <item>devanagari_1</item>
-    <item>devanagari_2</item>
-    <item>he_il_1452_1</item>
-    <item>he_il_1452_2</item>
-    <item>ar_pc</item>
-    <item>ar_alt</item>
+    <item>arab_ara_ibm</item>
+    <item>arab_ara_ibm_ma</item>
+    <item>beng_ben_rh</item>
+    <item>cyrl_bul_yaverti</item>
+    <item>cyrl_ukr_jcuken</item>
+    <item>cyrl_rus_jcuken</item>
+    <item>deva_hin_inscript</item>
+    <item>deva_hin_lv</item>
+    <item>grek_ell_qwerty</item>
+    <item>hebr_heb_si_1452_1</item>
+    <item>hebr_heb_si_1452_2</item>
+    <item>kore_kor_ks_x_5002</item>
```

Comment:
That's `kore_kor_dubeolsik_ks_x_5002` ?

Reply:
Yes. That was a mistake.

Reply:
The problem in `method.xml` is `qwertz_hu`. I have found the other `R.xml.qwerty`, too.